docker/vm/nodejs/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:8.11.2 as node
```

Comment:
Can we reuse the existing `Dockerfile` rather than creating a new one? It feels like a bad design decision since we would have to maintain multiple copies of the Dockerfile.

The way it should work is that we have one image for prod and that is what gets pushed to Docker Hub and which is what is used for the vm deployment as well.

Reply:
What changes do you  need to make for the prod one?